doc/source/examples/similarity_explanations_20ng.ipynb

```diff
@@ -0,0 +1,1029 @@
+{
```

Comment:
<p>Add "Utils" header + all the comments around spelling I made on the MNIST example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>